src/ports/postgres/modules/deep_learning/input_data_preprocessor.py_in

```diff
@@ -204,42 +207,90 @@ class InputDataPreprocessorDL(object):
         dep_shape = self._get_dependent_var_shape()
         dep_shape = ','.join([str(i) for i in dep_shape])
 
-        # Create the mini-batched output table
         if is_platform_pg():
             distributed_by_clause = ''
             dist_key_clause = ''
             join_clause = ''
             select_clause = 'b.*'
```

Comment:
I guess this was already in the code, but just happened to notice that this `select clause` isn't used anywhere.  I think we can just remove it.

Reply:
Documentation looks good, but there is also a usage message that needs to be updated.  (Lines 593-624 of input_data_preprocessor.py_in)